src/components/PlayerDash.jsx

```diff
@@ -30,11 +31,14 @@ module.exports = React.createClass({
     // console.log(this.state.players);
     const player = _.find(this.state.players, p => p.id == this.props.params.playerId);
     return (
-      <div className="Player">
-        { player
-          ? JSON.stringify(player)
-          : "not found"
-        }
+      <div className="container-fluid">
```

Comment:
`<div className="container-fluid">`
The container fluid class is on the root element in index.html so don't need to duplicate here.